### DISCUSSION TITLE
Permitted Chrome browser application not allowing Camera & Microphone automatically through the arguments in SEB v3.5.0

### DISCUSSION THREAD

**Discussion by Chris251290** [SafeExamBrowser/seb-win-refactoring#624] [Q&A]
Hi Team,

We are using "Windows 10" laptop with "SEB version 3.5.0 latest".

We have allowed the Chrome browser in permitted process inside the SEB Configuration and tried to launch the exam through Chrome browser. 

So, We have enabled the below argument  .
--auto-accept-camera-and-microphone-capture

We have taken this argument from the below link:
https://peter.sh/experiments/chromium-command-line-switches/

But, this argument not allowed the camera and microphone automatically. Once we disabled the argument, then only its asking to Allow/Block Camera & Microphone.

Kindly suggest if any alternate argument will work for automatic allowing camera and microphone. Or Let us know if we need to modify anythhing in the configuration.

Here I attached the screenshot for your reference.
![27062023_LIV_SEB (1)](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/111885140/b5f4a37b-7c55-4c6f-a5e2-452ddfb6e812)

**Comment by dbuechel** [comment]
Have you verified that the command-line switch works outside of SEB, i.e. when directly starting Chrome from the Windows command prompt?
